pkgs/tools/misc/ondir/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "ondir";
+  version = "0.2.2";
```

Comment:
Latest version, from 10 years ago is 0.2.3

Reply:
Thanks. Not sure how I missed that...